packages/mds-compliance-service/service/provider.ts

```diff
@@ -28,19 +28,40 @@ import {
 } from '../@types'
 import { ComplianceRepository } from '../repository'
 import { ComplianceViolationPeriodEntityToDomainCreate } from '../repository/mappers'
+import { ComplianceSnapshotStreamKafka } from './stream'
 import {
   ValidateComplianceSnapshotDomainModel,
   ValidateGetComplianceSnapshotsByTimeIntervalOptions
 } from './validators'
 
+export const getKafkaEnabled = (): boolean => {
+  return process.env.COMPLIANCE_KAFKA_STREAM !== 'false'
```

Comment:
I'm following the pattern I saw in `agtm-ab-service`. How do I ensure this env var is set properly when deployed?

Reply:
I would just always have this enabled, the toggle for ABT was so  Kafka could be skipped in the test suite without mocking.